homeassistant/components/zha/api.py

```diff
@@ -660,6 +672,38 @@ def async_load_api(hass):
         schema=SERVICE_SCHEMAS[SERVICE_ISSUE_ZIGBEE_CLUSTER_COMMAND],
     )
 
+    async def issue_zigbee_group_command(service):
+        """Issue command on zigbee cluster on a zigbee group."""
+        group_id = service.data.get(ATTR_GROUP)
+        cluster_id = service.data.get(ATTR_CLUSTER_ID)
+        command = service.data.get(ATTR_COMMAND)
+        args = service.data.get(ATTR_ARGS)
```

Comment:
Do we get `args` as a list or as a `str` as defined in schema? Have you tried issuing a cluster command, like `level.move_to_level(20, 100)`

Reply:
Changed to list and tested it with move_to_level.